src/Tables/molecules/TypeTable.tsx

```diff
@@ -49,29 +48,55 @@ interface IProps {
   hasStatus?: boolean
   hasThumbnail?: boolean
   hasTypeIcon?: boolean
+  initialSortColumnId?: boolean
   selectCallback? : (checked:boolean, id?: string | number)=>void
   toggleAllCallback? : (checked: boolean)=>void
+  sortCallback? : (ascending: boolean, columnId?: string) => void
```

Comment:
I updated the sortCallback with columnId as optional as we talked about it.
Because of that I'm handling the column toggles with the index value of the columns.